apps/api/R/models.R

```diff
@@ -19,21 +19,31 @@ getModel <- function(model_id, res){
   
   qry_res <- Model %>% collect()
   
-  PEcAn.DB::db.close(dbcon)
-  
   if (nrow(qry_res) == 0) {
+    PEcAn.DB::db.close(dbcon)
```

Comment:
You might want to just use `on.exit` to specify the db.close right when you open it to make sure that close happens even if any other sort of error occurs

Reply:
Yeah sure...that's a good idea...will try n incorporate this for all endpoints

Reply:
I think the problem is that the application does not exit, but stays open. We just need to make sure we close teh connections.